### PR TITLE
akita permissions removal #7285

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

**Comment by RemcoSimonides** [comment]
There is a proposal to remove 'super admin', 'admin', and 'member' btw
https://app.zenhub.com/workspaces/design---ux-60ab5f37bd1f990011c933a2/issues/blockframes/blockframes/6753

File: libs/organization/src/lib/organization/+state/organization.service.ts [hunk 1]
```diff
@@ -24,7 +24,13 @@ import { FireAnalytics } from '@blockframes/utils/analytics/app-analytics';
 export class OrganizationService extends CollectionService<OrganizationState> {
   readonly useMemorization = true;
 
-  private app = getCurrentApp(this.routerQuery)
+  private app = getCurrentApp(this.routerQuery);
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/+state/organization.service.ts:27]: you're not using it. Do you still need it ?

File: libs/organization/src/lib/permissions/+state/permissions.service.ts [hunk 2]
```diff
@@ -69,4 +105,14 @@ export class PermissionsService extends CollectionService<PermissionsState> {
...
     }
   }
+
+  /** Checks if the user is admin of his organization. */
+  public isUserAdmin(userId: string = this.auth.userId): boolean {
+    return this.permissions.roles[userId] === 'admin' || this.isUserSuperAdmin(userId);
+  }
+
+  /** Checks if the user is superAdmin of his organization. */
+  private isUserSuperAdmin(userId: string): boolean {
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/permissions/+state/permissions.service.ts:115]: you don't fallback to auth.userId ?

**Review by GrandSchtroumpf — APPROVED** [review]

File: libs/organization/src/lib/organization/+state/organization.service.ts (hunk 1 above)
Comment (phpgeekfr) [reply libs/organization/src/lib/organization/+state/organization.service.ts:27]: this.app is still used in the service

File: libs/organization/src/lib/permissions/+state/permissions.service.ts (hunk 2 above)
Comment (phpgeekfr) [reply libs/organization/src/lib/permissions/+state/permissions.service.ts:115]: this private function is only used by `isUserAdmin` that always provides userId

**Pull request merged**